BangazonAPI/BangazonAPI/Models/Product.cs

```diff
@@ -13,7 +13,7 @@ public class Product
 
         public int CustomerId { get; set; }
```

Comment:
This is more of a suggestion, and it doesn't have to be changed on this PR-- it's helpful to put a `[Required]` annotation above any properties that cannot be null in the database. Right now I can create a product and leave off its customer Id and I get a very crazy looking SQL error in Postman. If you put a `[Required]`, I could get a nice friendly error message instead 👍